README.md

```diff
@@ -874,7 +874,7 @@ A curated list of awesome C++ (or C) frameworks, libraries, resources, and shiny
 * [libvpx](http://www.webmproject.org/code/) - VP8/VP9 Codec SDK. [BSD]
 * [FFmpeg](https://www.ffmpeg.org/) - A complete, cross-platform solution to record, convert and stream audio and video. [LGPL2/GPL2]
 * [libde265](https://github.com/strukturag/libde265) - Open h.265 video codec implementation. [LGPL] [website](http://www.libde265.org/)
-* [x265](https://bitbucket.org/multicoreware/x265/wiki/Home) - Open h.265 video codec implementation. [GPL2] [website](https://x265.readthedocs.io/en/default/)
+* [x265](https://bitbucket.org/multicoreware/x265_git/src) - Open h.265 video codec implementation. [GPL2] [website](https://x265.readthedocs.io/en/master/)
```

Comment:
https://bitbucket.org/multicoreware/x265_git/wiki/Home might be a better 1:1 replacement

Reply:
Yes that is a better 1:1 replacement but I thought that the first link needed to be the source so that is why I changed it. I think they will both work so you can decide which one you want.